languages/javascript/reference/info/scope.md

```diff
@@ -0,0 +1,7 @@
+## Scope / Visibility
+
+Most languages have some concept of encapsulation through scope and visibility.
+Tracks should explain the language-concepts and keywords that their track uses (e.g. `public`, `private`, `export`) and what the normal way of using those scopes are in their language. An exercise may be appropriate, but providing examples in stubs early on may avoid overwhelming the student.
```

Comment:
I am not sure about this sentence (with the  `public`, `private`, `export`). What is this for? And aren't this more like Access modifiers? They do not define the scope.

I would rather suggest explaining what scope means (that it is about the boundary in which a variable is defined, or smth).

Reply:
This seems to be just copied from the global "scope" file.

Reply:
Indeed it is.

Reply:
Do we need it here?